bigframes/core/indexes/base.py

```diff
@@ -411,11 +406,52 @@ def fillna(self, value=None) -> Index:
             ops.fillna_op.as_expr(ex.free_var("arg"), ex.const(value))
         )
 
-    def rename(self, name: Union[str, Sequence[str]]) -> Index:
-        names = [name] if isinstance(name, str) else list(name)
```

Comment:
I think this change to Label (Hashable) is what's breaking some MultiIndex tests. Need a better way to check if something is already a sequence / iterable and not a string.

Reply:
Fixed in https://github.com/googleapis/python-bigquery-dataframes/pull/1744/commits/45f9232bfa93ba2d2cc6043c9d6e42c7f63dc4d9

I think we want to support integer names, too, so instead I specifically exclude tuple from the check for hashable objects.